src/sonic-containercfgd/tests/conftest.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+def init_env():
+    os.environ["CONTAINER_NAME"] = "2"
```

Comment:
Is there any significance to 2?

Reply:
No, here is just for making sure there is an env variable named "CONTAINER_NAME". I will change it to something different and add comment.